github/plugin.py

```diff
@@ -88,3 +89,128 @@ def github_pulls(
         for obj in response_json
     ]
     return pull_requests
+
+
+@dataclass
+class GitHubIssue:
```

Comment:
Rename `PullRequest` to `GitHubPullRequest` for consistency?  (GitHub does not let me comment on the `PullRequest` definition line.)

Reply:
Is there a concern about backwards compatibility?

Reply:
It can break things, as we remain in "chaos mode".  However, I think it's a reasonable scream test if it breaks.  In general, we haven't worried much about backwards compatibility, and I haven't heard of many external complaints.  (This probably changes day by day though as more people do things).